.travis.yml

```diff
@@ -16,8 +16,8 @@ addons:
       - libopencv-dev
 
 script:
-  - istanbul cover ./node_modules/mocha/bin/_mocha --report lcovonly -- "test/**/*_spec.js" --reporter spec --recursive
+  - nyc --reporter=lcov mocha --recursive
   - mv test/test_files/benchmark benchmark/testdata
-  - istanbul cover ./benchmark/image_preprocessors.js --report lcovonly --config ./benchmark/.istanbul.yml --  --only-gh-report opencv
+  - nyc --instrument=*/benchmark/*.js --reporter=json ./benchmark/image_preprocessors.js --only-gh-report opencv
```

Comment:
Why do you instrument the benchmark files? Previously, we wanted to exclude these files (now it's in `nyc.exclude`).